Demo/src/main/java/com/paypal/android/ui/approveorderprogress/ApproveOrderProgressFragment.kt

```diff
@@ -164,15 +154,17 @@ class ApproveOrderProgressFragment : Fragment() {
         val order = sdkSampleServerAPI.getOrder(orderId)
         viewModel.appendEventToLog(ApproveOrderEvent.GetOrder(order))
 
+        val payPalClientMetadataId = payPalDataCollector.getClientMetadataId(requireContext())
```

Comment:
Can we update our [docs](https://github.com/paypal/paypal-android/tree/main/docs) to show how to use this feature?

Reply:
That's what DTPPCPSDK-699 is actually for. When I started this ticket I noticed the duplicate environment enum though. I figured it would be better to fix the implementation in its own PR.

Reply:
Actually this is a good call–the docs won't make the PR much larger. I'll add them in to this PR.